client/components/game/imports/PuzzleAnswerForm.jsx

```diff
@@ -52,10 +52,12 @@ export default class PuzzleAnswerForm extends React.Component {
 
   _handleSubmit(e) {
     e.preventDefault();
-    const smartQuoteRegex = /\u{201c}|\u{201d}/ug
-    const smartApostropheRegex = /\u{2018}|\u{2019}/ug
+    // const smartQuoteRegex = /\u{201c}|\u{201d}/ug
+    // const smartApostropheRegex = /\u{2018}|\u{2019}/ug
+    const nonCharacterDigit = /[^a-zA-Z0-9]/ug
     const { puzzle } = this.props;
-    const answer = this.state.answer.replace(smartQuoteRegex, '"').replace(smartApostropheRegex, '\'');
+    // const answer = this.state.answer.replace(smartQuoteRegex, '"').replace(smartApostropheRegex, '\'');
+    const answer = this.state.answer.replaceAll(nonCharacterDigit, '');
```

Comment:
[no change needed - just out of curiosity] do we actually need this to be `replaceAll`? I wouldn't have expected as much since the regex ends in `g`

Reply:
The javascript docs seemed ambiguous on that point to me, so I went with replaceAll to be sure. It may not actually be needed, but I decided to play it safe.